src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -0,0 +1,23 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Entity(name = "PRODUCTS")
+public class Product {
```

Comment:
co to tu robi?

Reply:
żeby ustalić relację encji Group z encją Product, musiałem stworzyć podstawową encję Product. Inaczej projekt by się nie budował

Reply:
Próbowałem na maxa uprościć encję Product. Jedyna najprostsza wersja która poprawnie się buduje jest zawarta w ostatnim commitcie